.travis.yml

```diff
@@ -1,24 +1,60 @@
-language: python
-
-python:
-  - "3.6"
-
-install:
-  # Install conda
-  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
-  - conda clean --all -y
-  - ./python_env_build.sh
-  - source activate pb_env
-  - python manage.py collectstatic --no-input
-
-addons:
-    postgresql: "9.4"
-
-before_script:
-  - psql -c 'create database test_db;' -U postgres
-
-script:
-  - py.test webapp/apps/
+matrix:
+  include:
+  - language: ruby  # Cannot be 'minimal' in order to have PostgreSQL
+
+    sudo: required
+
+    services: docker
+
+    env: TAG=${TRAVIS_COMMIT::6}
+
+    # Need NEW_RELIC_TOKEN to be defined in Travis settings
+    install:
+      - make NEW_RELIC_TOKEN=$NEW_RELIC_TOKEN webapp-build
+      - pip install --user git-lint pylint pycodestyle
+      - export PATH=$PATH:/$HOME/.local/bin
+
+    addons:
+      postgresql: "9.4"
+
+    before_script: psql -c 'create database mypb;' -U postgres
+
+    script:
+      - >
+        docker run --net host -e PORT=80 -e DJANGO_SETTINGS_MODULE=webapp.settings
+        -e DROPQ_WORKERS=127.0.0.1:5050
+        -e DATABASE_URL=postgresql://postgres@localhost/mypb
+        -it opensourcepolicycenter/web:$TAG
+        /bin/bash -c "pip install -q pytest-django && py.test webapp/apps"
```

Comment:
Let's move `pytest-django` into the `requirements.txt` file. That way people can run these tests locally with ease.

Reply:
It's already in `requirements_dev.txt`, which gets installed by the `python_env_build.sh` script which is run by end-users. So I think it actually makes sense the current way, right? It probably shouldn't be in the image that gets deployed to e.g. Heroku.